modules/core/src/main/java/org/apache/ignite/internal/cdc/CdcMain.java

```diff
@@ -616,6 +652,18 @@ private void consumeSegmentActively(IgniteWalIteratorFactory.IteratorParametersB
         catch (IgniteCheckedException | IOException e) {
             throw new IgniteException(e);
         }
+        finally {
+            updateWalProcessingMetrics(System.nanoTime() - start);
+        }
+    }
+
+    /**
+     * Updates consumer event coonsumption metrics.
+     * @param duration - Operation durtion.
+     */
+    private void updateWalProcessingMetrics(long duration) {
```

Comment:
This method consisting of two lines is used only once, why should we add it?

Reply:
I added it for consistency between `eventsConsumptionTime` and `eventsConsumptionTimeTotal`. By calling this method we fix the duration as a method variable, and assign the same value to both metrics.